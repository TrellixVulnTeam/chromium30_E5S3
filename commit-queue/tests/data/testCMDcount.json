[["http://build.chromium.org/p/tryserver.chromium/json/builders/linux?filter=1","{\"state\":\"building\",\"basedir\":\"linux\",\"cachedBuilds\":[3245,3246,3247,3248,3249,3250,3251,3252,3253,3254],\"slaves\":[\"vm384-m4\",\"vm385-m4\",\"vm387-m4\",\"vm388-m4\",\"vm389-m4\",\"vm390-m4\",\"vm391-m4\",\"vm392-m4\",\"vm393-m4\",\"vm394-m4\",\"vm395-m4\",\"vm396-m4\",\"vm397-m4\",\"vm398-m4\",\"vm399-m4\",\"vm400-m4\",\"vm401-m4\",\"vm402-m4\",\"vm403-m4\",\"vm404-m4\",\"vm405-m4\",\"vm406-m4\",\"vm407-m4\",\"vm408-m4\",\"vm409-m4\",\"vm410-m4\",\"vm411-m4\",\"vm412-m4\",\"vm413-m4\",\"vm414-m4\",\"vm415-m4\",\"vm416-m4\",\"vm417-m4\",\"vm418-m4\",\"vm419-m4\",\"vm420-m4\",\"vm421-m4\",\"vm422-m4\",\"vm423-m4\",\"vm424-m4\",\"vm425-m4\",\"vm426-m4\",\"vm427-m4\",\"vm429-m4\",\"slave1-b1\",\"slave2-b1\",\"slave3-b1\",\"slave4-b1\",\"slave5-b1\",\"slave6-b1\",\"slave7-b1\",\"slave8-b1\",\"slave9-b1\",\"slave10-b1\",\"slave11-b1\",\"slave12-b1\",\"slave13-b1\",\"slave14-b1\",\"slave15-b1\",\"slave16-b1\",\"slave17-b1\",\"slave18-b1\",\"slave19-b1\",\"slave20-b1\",\"slave21-b1\",\"slave22-b1\",\"slave23-b1\",\"slave24-b1\",\"slave25-b1\",\"slave26-b1\",\"slave27-b1\",\"slave28-b1\",\"slave29-b1\",\"slave30-b1\",\"slave31-b1\",\"slave32-b1\",\"slave33-b1\",\"slave34-b1\",\"slave35-b1\",\"slave36-b1\",\"slave37-b1\",\"slave38-b1\",\"slave39-b1\",\"slave40-b1\",\"slave41-b1\",\"slave42-b1\",\"slave43-b1\",\"slave44-b1\",\"slave45-b1\",\"slave46-b1\",\"slave47-b1\",\"slave48-b1\",\"slave49-b1\",\"slave50-b1\",\"slave51-b1\",\"slave52-b1\",\"slave53-b1\",\"slave54-b1\",\"slave55-b1\",\"slave56-b1\",\"slave57-b1\",\"slave58-b1\",\"slave59-b1\",\"slave60-b1\",\"slave61-b1\",\"slave62-b1\",\"slave63-b1\",\"slave64-b1\",\"slave65-b1\",\"slave66-b1\",\"slave67-b1\",\"slave68-b1\",\"slave69-b1\",\"slave70-b1\",\"slave71-b1\",\"slave72-b1\",\"slave73-b1\",\"slave74-b1\",\"slave75-b1\",\"slave76-b1\",\"slave77-b1\",\"slave78-b1\",\"slave79-b1\",\"slave80-b1\",\"slave81-b1\",\"slave82-b1\",\"slave83-b1\",\"slave84-b1\",\"slave85-b1\",\"slave86-b1\",\"slave87-b1\",\"slave88-b1\",\"slave89-b1\",\"slave90-b1\",\"slave91-b1\",\"slave92-b1\",\"slave93-b1\",\"slave94-b1\",\"slave95-b1\",\"slave96-b1\",\"slave97-b1\",\"slave98-b1\",\"slave99-b1\",\"slave100-b1\"],\"currentBuilds\":[3248,3249,3252,3253,3254]}"],["http://build.chromium.org/p/tryserver.chromium/json/builders/linux/builds/3254?filter=1","{\"slave\":\"vm419-m4\",\"builderName\":\"linux\",\"number\":3254,\"currentStep\":{\"name\":\"update\",\"step_number\":1,\"expectations\":[[\"output\",170138,152441.25]],\"isStarted\":true,\"times\":[1348166168.92174,null],\"text\":[\"updating\",\"rHEAD\"]},\"times\":[1348166164.117495,null],\"blame\":[\"maruel@chromium.org\"],\"reason\":\"8897024-29003: .\",\"eta\":4881.735800027847,\"steps\":[{\"name\":\"update_scripts\",\"isFinished\":true,\"text\":[\"update_scripts\"],\"expectations\":[[\"output\",3201,3700.0]],\"isStarted\":true,\"times\":[1348166164.125098,1348166168.920582]},{\"name\":\"update\",\"step_number\":1,\"expectations\":[[\"output\",170138,152441.25]],\"isStarted\":true,\"times\":[1348166168.92174,null],\"text\":[\"updating\",\"rHEAD\"]},{\"step_number\":2,\"expectations\":[[\"output\",null,1280.25]],\"name\":\"apply_issue\"},{\"step_number\":3,\"expectations\":[[\"output\",null,2388.5]],\"name\":\"runhooks\"},{\"step_number\":4,\"expectations\":[[\"output\",null,947.25]],\"name\":\"cleanup_temp\"},{\"step_number\":5,\"expectations\":[[\"output\",null,1355619.25]],\"name\":\"compile\"},{\"step_number\":6,\"expectations\":[[\"output\",null,null]],\"name\":\"manifests_to_hashes\"},{\"step_number\":7,\"expectations\":[[\"output\",null,null]],\"name\":\"trigger_swarm_tests\"},{\"step_number\":8,\"expectations\":[[\"output\",null,145451.0]],\"name\":\"interactive_ui_tests\"},{\"step_number\":9,\"expectations\":[[\"output\",null,8797.0]],\"name\":\"check_deps2git\"},{\"step_number\":10,\"expectations\":[[\"output\",null,1980.25]],\"name\":\"check_deps2submodules\"},{\"step_number\":11,\"expectations\":[[\"output\",null,1620.25]],\"name\":\"check_deps\"},{\"step_number\":12,\"expectations\":[[\"output\",null,null]],\"name\":\"check_licenses\"},{\"step_number\":13,\"expectations\":[[\"output\",null,220724.5]],\"name\":\"base_unittests\"},{\"step_number\":14,\"expectations\":[[\"output\",null,133263.5]],\"name\":\"cacheinvalidation_unittests\"},{\"step_number\":15,\"expectations\":[[\"output\",null,207604.0]],\"name\":\"cc_unittests\"},{\"step_number\":16,\"expectations\":[[\"output\",null,27783.75]],\"name\":\"crypto_unittests\"},{\"step_number\":17,\"expectations\":[[\"output\",null,18507.5]],\"name\":\"dbus_unittests\"},{\"step_number\":18,\"expectations\":[[\"output\",null,null]],\"name\":\"googleurl_unittests\"},{\"step_number\":19,\"expectations\":[[\"output\",null,232286.0]],\"name\":\"gpu_unittests\"},{\"step_number\":20,\"expectations\":[[\"output\",null,44979.0]],\"name\":\"jingle_unittests\"},{\"step_number\":21,\"expectations\":[[\"output\",null,173816.25]],\"name\":\"media_unittests\"},{\"step_number\":22,\"expectations\":[[\"output\",null,596074.0]],\"name\":\"net_unittests\"},{\"step_number\":23,\"expectations\":[[\"output\",null,15289.0]],\"name\":\"printing_unittests\"},{\"step_number\":24,\"expectations\":[[\"output\",null,67336.0]],\"name\":\"remoting_unittests\"},{\"step_number\":25,\"expectations\":[[\"output\",null,21395.5]],\"name\":\"test_shell_tests\"},{\"step_number\":26,\"expectations\":[[\"output\",null,19590.5]],\"name\":\"safe_browsing_tests\"},{\"step_number\":27,\"expectations\":[[\"output\",null,5607.25]],\"name\":\"sandbox_linux_unittests\"},{\"step_number\":28,\"expectations\":[[\"output\",null,14190.25]],\"name\":\"ipc_tests\"},{\"step_number\":29,\"expectations\":[[\"output\",null,181300.5]],\"name\":\"sync_unit_tests\"},{\"step_number\":30,\"expectations\":[[\"output\",null,1625061.25]],\"name\":\"unit_tests\"},{\"step_number\":31,\"expectations\":[[\"output\",null,7597.75]],\"name\":\"sql_unittests\"},{\"step_number\":32,\"expectations\":[[\"output\",null,42749.5]],\"name\":\"ui_unittests\"},{\"step_number\":33,\"expectations\":[[\"output\",null,352261.25]],\"name\":\"content_unittests\"},{\"step_number\":34,\"expectations\":[[\"output\",null,3275396.0]],\"name\":\"browser_tests\"},{\"step_number\":35,\"expectations\":[[\"output\",null,264534.75]],\"name\":\"content_browsertests\"},{\"step_number\":36,\"expectations\":[[\"output\",null,40889.5]],\"name\":\"webkit_compositor_bindings_unittests\"},{\"step_number\":37,\"expectations\":[[\"output\",null,null]],\"name\":\"sync_integration_tests\"},{\"step_number\":38,\"expectations\":[[\"output\",null,null]],\"name\":\"pyauto_functional_tests\"},{\"step_number\":39,\"expectations\":[[\"output\",null,659946.75]],\"name\":\"nacl_integration\"}],\"sourceStamp\":{\"project\":\"chrome\",\"changes\":[{\"rev\":\"HEAD\",\"who\":\"maruel@chromium.org\",\"when\":1348166144,\"number\":112715,\"at\":\"Thu 20 Sep 2012 11:35:44\",\"revision\":\"HEAD\"}],\"revision\":\"HEAD\"},\"properties\":[[\"blamelist\",[\"maruel@chromium.org\"],\"Build\"],[\"branch\",null,\"Build\"],[\"buildername\",\"linux\",\"Builder\"],[\"buildnumber\",3254,\"Build\"],[\"issue\",8897024,\"Try job\"],[\"mastername\",\"tryserver.chromium\",\"master.cfg\"],[\"patchset\",29003,\"Try job\"],[\"primary_repo\",\"\",\"Source\"],[\"project\",\"chrome\",\"Build\"],[\"repository\",\"\",\"Build\"],[\"revision\",\"HEAD\",\"Build\"],[\"rietveld\",\"https://chromiumcodereview.appspot.com/status_listener\",\"Try job\"],[\"root\",\"src\",\"Try job\"],[\"scheduler\",\"try_job_rietveld\",\"Scheduler\"],[\"slavename\",\"vm419-m4\",\"BuildSlave\"],[\"testfilter\",[\"defaulttests\"],\"Try job\"],[\"try_job_key\",\"ahdzfmNocm9taXVtY29kZXJldmlldy1ocnIyCxIFSXNzdWUYgISfBAwLEghQYXRjaFNldBjL4gEMCxIMVHJ5Sm9iUmVzdWx0GKKZAgw\",\"Try job\"],[\"workdir\",\"/mnt/data/b/build/slave/linux\",\"slave\"]]}"],["http://build.chromium.org/p/tryserver.chromium/json/builders/linux/builds/3253?filter=1","{\"slave\":\"vm405-m4\",\"builderName\":\"linux\",\"number\":3253,\"currentStep\":{\"name\":\"check_deps\",\"step_number\":11,\"expectations\":[[\"output\",1498,1620.25]],\"isStarted\":true,\"times\":[1348166285.731871,null],\"eta\":4.319195439180575,\"text\":[\"running check_deps\"]},\"times\":[1348164729.359081,null],\"blame\":[\"stevenjb@chromium.org\"],\"reason\":\"10947046-4001: Explicitly remove notifications before re-showing them.\",\"eta\":3139.0634150505066,\"steps\":[{\"name\":\"update_scripts\",\"isFinished\":true,\"text\":[\"update_scripts\"],\"expectations\":[[\"output\",3201,3700.0]],\"isStarted\":true,\"times\":[1348164729.362518,1348164732.673934]},{\"name\":\"update\",\"isFinished\":true,\"step_number\":1,\"expectations\":[[\"output\",148937,152441.25]],\"isStarted\":true,\"times\":[1348164732.675269,1348164837.366555],\"text\":[\"update\",\"r157752\",\"webkit r129100\",\"nacl r9768\",\"v8 r12514\"]},{\"name\":\"apply_issue\",\"isFinished\":true,\"step_number\":2,\"expectations\":[[\"output\",2427,1280.25]],\"isStarted\":true,\"times\":[1348164837.367549,1348164848.483012],\"text\":[\"apply patch\"]},{\"name\":\"runhooks\",\"isFinished\":true,\"step_number\":3,\"expectations\":[[\"output\",2808,2388.5]],\"isStarted\":true,\"times\":[1348164848.484293,1348164916.15593],\"text\":[\"gclient hooks\"]},{\"name\":\"cleanup_temp\",\"isFinished\":true,\"step_number\":4,\"expectations\":[[\"output\",894,947.25]],\"isStarted\":true,\"times\":[1348164916.157172,1348164916.334846],\"text\":[\"cleanup_temp\"]},{\"name\":\"compile\",\"isFinished\":true,\"step_number\":5,\"expectations\":[[\"output\",1142833,1355619.25]],\"isStarted\":true,\"times\":[1348164916.335957,1348165978.006808],\"text\":[\"compile\"]},{\"name\":\"manifests_to_hashes\",\"isFinished\":true,\"step_number\":6,\"results\":[3,null],\"isStarted\":true,\"expectations\":[[\"output\",null,null]],\"text\":[\"manifests_to_hashes\",\"skipped\"],\"times\":[1348165978.007779,1348165978.759594]},{\"name\":\"trigger_swarm_tests\",\"isFinished\":true,\"step_number\":7,\"results\":[3,null],\"isStarted\":true,\"expectations\":[[\"output\",null,null]],\"text\":[\"trigger_swarm_tests\",\"skipped\"],\"times\":[1348165978.760393,1348165979.919687]},{\"name\":\"interactive_ui_tests\",\"isFinished\":true,\"step_number\":8,\"expectations\":[[\"output\",140745,145451.0]],\"isStarted\":true,\"times\":[1348165980.01411,1348166278.358225],\"text\":[\"interactive_ui_tests\",\"2 flaky\"]},{\"name\":\"check_deps2git\",\"isFinished\":true,\"step_number\":9,\"expectations\":[[\"output\",8720,8797.0]],\"isStarted\":true,\"times\":[1348166278.359448,1348166285.260302],\"text\":[\"check_deps2git\"]},{\"name\":\"check_deps2submodules\",\"isFinished\":true,\"step_number\":10,\"expectations\":[[\"output\",1927,1980.25]],\"isStarted\":true,\"times\":[1348166285.261463,1348166285.731096],\"text\":[\"check_deps2submodules\"]},{\"name\":\"check_deps\",\"step_number\":11,\"expectations\":[[\"output\",1498,1620.25]],\"isStarted\":true,\"times\":[1348166285.731871,null],\"eta\":4.319195439180575,\"text\":[\"running check_deps\"]},{\"step_number\":12,\"expectations\":[[\"output\",null,null]],\"name\":\"check_licenses\"},{\"step_number\":13,\"expectations\":[[\"output\",null,220724.5]],\"name\":\"base_unittests\"},{\"step_number\":14,\"expectations\":[[\"output\",null,133263.5]],\"name\":\"cacheinvalidation_unittests\"},{\"step_number\":15,\"expectations\":[[\"output\",null,207604.0]],\"name\":\"cc_unittests\"},{\"step_number\":16,\"expectations\":[[\"output\",null,27783.75]],\"name\":\"crypto_unittests\"},{\"step_number\":17,\"expectations\":[[\"output\",null,18507.5]],\"name\":\"dbus_unittests\"},{\"step_number\":18,\"expectations\":[[\"output\",null,null]],\"name\":\"googleurl_unittests\"},{\"step_number\":19,\"expectations\":[[\"output\",null,232286.0]],\"name\":\"gpu_unittests\"},{\"step_number\":20,\"expectations\":[[\"output\",null,44979.0]],\"name\":\"jingle_unittests\"},{\"step_number\":21,\"expectations\":[[\"output\",null,173816.25]],\"name\":\"media_unittests\"},{\"step_number\":22,\"expectations\":[[\"output\",null,596074.0]],\"name\":\"net_unittests\"},{\"step_number\":23,\"expectations\":[[\"output\",null,15289.0]],\"name\":\"printing_unittests\"},{\"step_number\":24,\"expectations\":[[\"output\",null,67336.0]],\"name\":\"remoting_unittests\"},{\"step_number\":25,\"expectations\":[[\"output\",null,21395.5]],\"name\":\"test_shell_tests\"},{\"step_number\":26,\"expectations\":[[\"output\",null,19590.5]],\"name\":\"safe_browsing_tests\"},{\"step_number\":27,\"expectations\":[[\"output\",null,5607.25]],\"name\":\"sandbox_linux_unittests\"},{\"step_number\":28,\"expectations\":[[\"output\",null,14190.25]],\"name\":\"ipc_tests\"},{\"step_number\":29,\"expectations\":[[\"output\",null,181300.5]],\"name\":\"sync_unit_tests\"},{\"step_number\":30,\"expectations\":[[\"output\",null,1625061.25]],\"name\":\"unit_tests\"},{\"step_number\":31,\"expectations\":[[\"output\",null,7597.75]],\"name\":\"sql_unittests\"},{\"step_number\":32,\"expectations\":[[\"output\",null,42749.5]],\"name\":\"ui_unittests\"},{\"step_number\":33,\"expectations\":[[\"output\",null,352261.25]],\"name\":\"content_unittests\"},{\"step_number\":34,\"expectations\":[[\"output\",null,3275396.0]],\"name\":\"browser_tests\"},{\"step_number\":35,\"expectations\":[[\"output\",null,264534.75]],\"name\":\"content_browsertests\"},{\"step_number\":36,\"expectations\":[[\"output\",null,40889.5]],\"name\":\"webkit_compositor_bindings_unittests\"},{\"step_number\":37,\"expectations\":[[\"output\",null,null]],\"name\":\"sync_integration_tests\"},{\"step_number\":38,\"expectations\":[[\"output\",null,null]],\"name\":\"pyauto_functional_tests\"},{\"step_number\":39,\"expectations\":[[\"output\",null,659946.75]],\"name\":\"nacl_integration\"}],\"sourceStamp\":{\"project\":\"chrome\",\"changes\":[{\"rev\":\"157752\",\"who\":\"stevenjb@chromium.org\",\"when\":1348164728,\"number\":112637,\"at\":\"Thu 20 Sep 2012 11:12:08\",\"revision\":\"157752\"}],\"revision\":\"157752\"},\"properties\":[[\"blamelist\",[\"stevenjb@chromium.org\"],\"Build\"],[\"branch\",null,\"Build\"],[\"buildername\",\"linux\",\"Builder\"],[\"buildnumber\",3253,\"Build\"],[\"got_nacl_revision\",\"9768\",\"Source\"],[\"got_revision\",\"157752\",\"Source\"],[\"got_v8_revision\",\"12514\",\"Source\"],[\"got_webkit_revision\",\"129100\",\"Source\"],[\"gtest_filter\",null,\"Factory\"],[\"issue\",10947046,\"Try job\"],[\"mastername\",\"tryserver.chromium\",\"master.cfg\"],[\"patchset\",4001,\"Try job\"],[\"primary_repo\",\"\",\"Source\"],[\"project\",\"chrome\",\"Build\"],[\"repository\",\"\",\"Build\"],[\"revision\",\"157752\",\"Build\"],[\"rietveld\",\"https://chromiumcodereview.appspot.com/status_listener\",\"Try job\"],[\"root\",\"src\",\"Try job\"],[\"scheduler\",\"try_job_rietveld\",\"Scheduler\"],[\"slavename\",\"vm405-m4\",\"BuildSlave\"],[\"swarm_tests\",\"\",\"Step\"],[\"testfilter\",null,\"Try job\"],[\"try_job_key\",\"ahdzfmNocm9taXVtY29kZXJldmlldy1ocnIwCxIFSXNzdWUY5pOcBQwLEghQYXRjaFNldBihHwwLEgxUcnlKb2JSZXN1bHQY8S4M\",\"Try job\"],[\"warnings-count\",0,\"WarningCountingShellCommand\"],[\"workdir\",\"/mnt/data/b/build/slave/linux\",\"slave\"]]}"]]