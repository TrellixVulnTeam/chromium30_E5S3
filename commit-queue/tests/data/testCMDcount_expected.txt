"linux\n.. 1\nlinux: 1\nTotal: 1\n"